cranelift-codegen/meta/src/isa/x86/encodings.rs

```diff
@@ -59,8 +59,8 @@ impl PerCpuModeEncodings {
         template: Template,
         builder_closure: T,
     ) -> Encoding
-    where
-        T: FnOnce(EncodingBuilder) -> EncodingBuilder,
+        where
+            T: FnOnce(EncodingBuilder) -> EncodingBuilder,
```

Comment:
Should I be running `rustfmt` on everything to avoid this?

Reply:
Travis to the rescue: https://travis-ci.org/CraneStation/cranelift/jobs/556043281#L1455--`cargo fmt --all`.

Reply:
Yes :-). For example, I have my editor set up to run `rustfmt` automatically, so it happens automatically.

Reply:
👍